src/lexers/javascript-lexer.js

```diff
@@ -10,35 +10,32 @@ const WalkerBase = Object.assign({}, walk.base, {
   }
 })
 
+const defaultInjectors = [injectAcornEs7, injectAcornStage3]
+const defaultPlugins = { stage3: true, es7: true }
+
 export default class JavascriptLexer extends BaseLexer {
   constructor(options = {}) {
     super(options)
 
-    this.acornOptions = { 
-      sourceType: 'module', 
+    this.acornOptions = {
+      sourceType: 'module',
       ecmaVersion: 9,
       ...options.acorn,
       plugins: {
-        stage3: true,
-        es7: true,
+        ...defaultPlugins,
         ...(options.acorn ? options.acorn.plugins : {})
       }
     }
 
     this.functions = options.functions || ['t']
     this.attr = options.attr || 'i18nKey'
+    this.injectors = [...defaultInjectors, ...((options.acorn && options.acorn.injectors) || [])]
```

Comment:
Could become `this.injectors = options.acorn && options.acorn.injectors || []` if we remove the defaults

Reply:
Yep